tslint-sql.json

```diff
@@ -13,19 +13,22 @@
 				"argIndex": 1
 			}
 		],
-		"no-floating-promises":{
+		"no-floating-promises": {
 			"severity": "warn"
 		},
 		"await-promise": {
 			"severity": "warn",
-			"options": ["Thenable"]
+			"options": [
+				"Thenable"
+			]
 		},
-		"no-promise-as-boolean": {
```

Comment:
Purposely removing this?

Reply:
Yeah, our tslint is out of date for that rule.